### PR TITLE
Small typo fix

### PR DESCRIPTION
**Pull request by whaber**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -180,7 +180,7 @@ Membership governance can be viewed [here](https://github.com/cncf/sig-security/
 * Tabitha Sable ([@tabbysable](https://github.com/tabbysable))
 * Steven Hadfield ([@steven-hadfield](https://github.com/steven-hadfield)), FICO
 * Yeeling Lam ([@yeelinglam](https://github.com/yeelinglam)), AT&T
-* Wayne Haber ([@whaber github](https://github.com/whaber) / ([@whaber gitlab](https://gitlab.com/whaber), GitLab
+* Wayne Haber ([@whaber github](https://github.com/whaber) / ([@whaber gitlab](https://gitlab.com/whaber)), GitLab
```
[the remark this answers is not in the document]
Comment (whaber) [reply README.md:183]: Much better @lumjjb

[the remark this answers is not in the document]
Comment (lumjjb) [reply README.md:183]: @whaber I believe the typo is still in the branch, could you fix it and I can merge it - tks!

[the remark this answers is not in the document]
Comment (whaber) [reply README.md:183]: @lumjjb I don't see the typo still in the branch.  Can you confirm?

[the remark this answers is not in the document]
Comment (lumjjb) [reply README.md:183]: There seems to be an additional "(" just before the gitlab ref link

* Wayne Haber ([@whaber github](https://github.com/whaber) / ([@whaber gitlab](https://gitlab.com/whaber)), GitLab

[the remark this answers is not in the document]
Comment (whaber) [reply README.md]: Got it. Fixed: https://github.com/whaber/sig-security/blob/patch-2/README.md

[the remark this answers is not in the document]
Comment (lumjjb) [reply README.md]: Thanks!

**Pull request merged**